bench/irmin-pack/layers.ml

```diff
@@ -51,6 +51,11 @@ module Store =
     (Irmin.Branch.String)
     (Hash)
 
+let info () =
+  let date = Int64.of_float (Unix.gettimeofday ()) in
```

Comment:
```suggestion
  let date = 0L in
```

Even if it never matters, it seems wrong to have `gettimeofday` in benchmarks.

Reply:
Fixed in #1405